docs/api/rest_api/rest_api_reference/input/ez-product-products.raml

```diff
@@ -156,18 +156,14 @@
                     Accept:
                         description: If set, gets price in given currency in JSON format.
                         example:  |
-                            application/json
-                    Content-Type:
-                        description: If set, the updated price is returned in JSON format.
-                        example:  |
-                            application/json
+                            application/vnd.ibexa.api.Price+json
```

Comment:
What about CustomPrice? Maybe application/json was there because the type of the request result is unknown.